src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -133,6 +141,11 @@ class PropertyAccessor implements PropertyAccessorInterface
     private $cacheItemPool;
 
     /**
+     * @var MetadataFactoryInterface
```

Comment:
It is really necessary? Then it should be also added to `$classMetadataFactory` above...